EdvClient.js

```diff
@@ -54,6 +56,10 @@ export class EdvClient {
     this.cipher = new Cipher();
     this.indexHelper = new IndexHelper();
     this.httpsAgent = httpsAgent;
+    this.defaultHeaders = {
+      ...defaultHeaders,
+      ...DEFAULT_HEADERS
```

Comment:
Is it best to overwrite provided defaults with library defaults?  Seems like it could lead to unexpected behavior.  Maybe better to just let people overwrite "Accept" and deal with the consequences of doing it wrong?

Reply:
I'm ok with @davidlehn's suggestion, so destructure `DEFAULT_HEADERS` first then `defaultHeaders`.

Reply:
I tend to agree, I did it this way based on the discussion in the issue. In this case, because `{Accept: 'application/ld+json, application/json'}` is rare, i think its probably helpful... of course if the server starts to support other formats this will need to change.

Reply:
I have implemented the suggestion: `{...DEFAULT_HEADERS, ...headers}`